### PR TITLE
seq seed and migrate change

### PR DESCRIPTION
**Pull request by essgje**
# \<Description\>

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

**Review by tommigitgaur — DISMISSED** [review]
Fine

File: workspace.json [hunk 1]
```diff
@@ -3804,6 +3804,27 @@
             "command": "yarn ts-node -P apps/skilavottord/ws/tsconfig.app.json apps/skilavottord/ws/src/buildSchema.ts"
           }
         },
+        "migrate": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "../../../node_modules/.bin/sequelize-cli db:migrate",
```
Comment (pshomov) [inline workspace.json:3810]: you should skip the `../../../node_modules/.bin` part, here and your other changes

**Review by pshomov — CHANGES_REQUESTED** [review]

File: workspace.json (hunk 1 above)
Comment (essgje) [reply workspace.json:3810]: Has this changed. this is same as in judical-system.?

Comment (essgje) [reply workspace.json]: I have done the changes.

**Review by pshomov — APPROVED** [review]

**Comment by pshomov** [comment]
LGTM

**Pull request merged**